cross/nano/Makefile

```diff
@@ -12,5 +12,6 @@ COMMENT  = nano is a text editor for Unix-like computing systems or operating en
 LICENSE  = GPLv2
 
 GNU_CONFIGURE = 1
+CONFIGURE_ARGS += --prefix=$(INSTALL_PREFIX)
```

Comment:
This line is obsolete and included with `GNU_CONFIGURE = 1`
```suggestion
```

Reply:
Thanks I thought it might be but wasn't sure.

Reply:
I recently found --prefix in cross/nodejs_v16/Makefile (where GNU_CONFIGURE must not be defined).